src/diffusers/loaders/lora_pipeline.py

```diff
@@ -1250,13 +1258,17 @@ def load_lora_weights(
         if not is_correct_format:
             raise ValueError("Invalid LoRA checkpoint.")
 
-        self.load_lora_into_transformer(
-            state_dict,
-            transformer=getattr(self, self.transformer_name) if not hasattr(self, "transformer") else self.transformer,
-            adapter_name=adapter_name,
-            _pipeline=self,
-            low_cpu_mem_usage=low_cpu_mem_usage,
-        )
+        transformer_state_dict = {k: v for k, v in state_dict.items() if "transformer." in k}
```

Comment:
For my understanding: This is a fix independent of the main change of this PR, right? Would it be possible to move this check inside of `load_lora_into_transformer` or would that not be a good idea?

Reply:
I think it kind of depends on the entrypoint to the underlying method. 

We already have a similar check within `load_lora_adapter()`:
https://github.com/huggingface/diffusers/blob/e187b706a546a54c95abffb951f0ec0292400627/src/diffusers/loaders/peft.py#L153

So, I think it should be fine, without.